packages/vuetify/src/components/VDataTable/VDataTable.ts

```diff
@@ -355,7 +355,14 @@ export default VDataIterator.extend({
 
       if (this.$scopedSlots['group.summary']) {
         children.push(this.$createElement('template', { slot: 'column.summary' }, [
-          this.$scopedSlots['group.summary']!({ group, groupBy: props.options.groupBy, items, headers: this.computedHeaders }),
+          this.$scopedSlots['group.summary']!({
```

Comment:
While I agree with you in terms of code styling, that is out of scope of this PR. You are welcome to make that update in a different PR.

Reply:
Changed it back to a single line